proposals/description/fip_999.ts

```diff
@@ -0,0 +1,48 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_999: ProposalDescription = {
+  title: 'FIP-XYZ: ------------------------',
+  commands: [
+    {
+      target: 'aaveEthPCVDeposit',
+      values: '0',
+      method: 'withdraw(address,uint256)',
+      arguments: ['{balancerDepositBalWeth}', '200000000000000000000'],
+      description: 'Withdraw 200 WETH from Aave to the BAL/WETH PCVDeposit'
+    },
+    {
+      target: 'bal',
+      values: '0',
+      method: 'transfer(address,uint256)',
+      arguments: ['{balancerDepositBalWeth}', '200000000000000000000000'],
+      description: 'Move 200k BAL from Timelock to the deposit'
+    },
+    {
+      target: 'balancerDepositBalWeth',
+      values: '0',
+      method: 'deposit()',
+      arguments: [],
+      description: 'Deposit BAL and WETH in the Balancer pool'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'swapDeposit(address,address)',
+      arguments: ['{balDepositWrapper}', '{balancerDepositBalWeth}'],
```

Comment:
Should we add a WETH lens?

Reply:
I guess fully reporting in BAL is cool

Reply:
Spicier: what will we do for the 80% TRIBE 20% WETH pool ?